src/main/java/org/jitsi/videobridge/cc/vp8/VP8Frame.java

```diff
@@ -327,7 +327,8 @@ boolean decodes(@NotNull VP8Frame vp8Frame)
 
             // XXX this actually makes it possible to miss a reference frame :(
             // It would require significant effort to prevent such a case.
-            return isNextTL0(vp8Frame) || matchesTL0PICIDX(vp8Frame);
+            return isNextTemporalBaseLayerFrame(vp8Frame)
+                || matchesTL0PICIDX(vp8Frame);
```

Comment:
maybe this one should be `isCurrentTemporalBaseLayerFrame` to match?  then it will read very clearly:

`return isNextTemporalBaseLayerFrame(vp8Frame) || isCurrentTemporalBaseLayerFrame(vp8Frame)`

which would be awesome

Reply:
isCurrentTemporalBaseLayerFrame sounds a little bit misleading, can I suggest dependsOnSameTemporalBaseLayerFrame? My suggestion is too long, so I'm not too happy about it either.

Reply:
I'm fine with `dependsOnSameTemporalBaseLayerFrame`, I'd rather have it be long and descriptive than somewhat cryptic, so that's fine with me.